server/express.js

```diff
@@ -5,9 +5,9 @@ const morgan = require('morgan');
 
 const app = express();
 
-app.use(morgan('dev'));
 
-app.use(express.static(path.join(__dirname, '../public')));
+app.use(express.static(path.join(__dirname, '../client/dist')));
+app.use(morgan('dev'));
```

Comment:
Why are you changing the server file? This change should be reverted? The line for morgan is intentionally above the express.static line.